### DISCUSSION TITLE
How to align fastfetch output for key separator and values

### DISCUSSION THREAD

**Discussion by cristian-ciobanu** [fastfetch-cli/fastfetch#761] [Q&A] [answered]
I have this setup in neofetch

![image](https://github.com/fastfetch-cli/fastfetch/assets/7124193/b705ee6a-e357-4fad-ba9f-9324e6f2d25e)

 I'm trying to replicate the same setup using fastfetch but I'm unable to align the separator character on the right side.
Am I missing something or it cannot be done in the latest version yet ?

Thank you.

**Comment by CarterLi** [comment, marked as answer]
I created a new example https://github.com/fastfetch-cli/fastfetch/blob/dev/presets/examples/12.jsonc for you to refer to.

Screenshot:

![image](https://github.com/fastfetch-cli/fastfetch/assets/6134068/39e5b604-4b3d-4aaf-ad0b-dabe2925d588)

**Reply by cristian-ciobanu** [reply]
Thank you it works fine. Still, one small question regarding the keyColor parameter.
The number of colors is it limited to the 16 standard colors defined by the shell or it can support up to 256 colors ?

**Reply by CarterLi** [reply]
It uses the standard ansi escape codes. You may refer to `fastfetch -h color`
